redux/redux.d.ts

```diff
@@ -1,52 +1,81 @@
-// Type definitions for Redux v1.0.0
-// Project: https://github.com/rackt/redux
-// Definitions by: William Buchwalter <https://github.com/wbuchwalter/>, Vincent Prouillet <https://github.com/Keats/>
+// Type definitions for Redux v3.3.1
+// Project: https://github.com/reactjs/redux
+// Definitions by: Jakub Strojewski <https://github.com/ulfryk>
 // Definitions: https://github.com/borisyankov/DefinitelyTyped
 
 declare module Redux {
 
-    interface ActionCreator extends Function {
-        (...args: any[]): any;
+    // Interfaces
+
+    interface IAction {
```

Comment:
http://www.typescriptlang.org/Handbook#writing-dts-files

> Naming Conventions
In general, do not prefix interfaces with I (e.g. IColor). Because the concept of an interface in TypeScript is much more broad than in C# or Java, the IFoo naming convention is not broadly useful.